docs/pages/api-reference/liveblocks-react.mdx

```diff
@@ -1102,6 +1102,199 @@ notifications are not part of rooms, and errors for these will show. See the
 
 </Banner>
 
+## AI Copilots
+
+### useAiChats [@badge=LiveblocksProvider]
+
+Returns a paginated list of AI chats created by the current user. Initially
+fetches the latest 50 chats.
+
+```tsx
+import { useAiChats } from "@liveblocks/react";
+
+const { chats, error, isLoading } = useAiChats();
+```
+
+##### List the user’s chats and switch between them
+
+You can use the [`AiChat`](/docs/api-reference/liveblocks-react-ui#AiChat)
+component alongside the hook to create an AI chat switcher. Below, each button
+displays the chat’s automatically generated title, and chats can be deleted with
+[`useDeleteAiChat`](#useDeleteAiChat).
+
+```tsx
+import { useState } from "react";
+import { AiChat } from "@liveblocks/react-ui";
+import { useAiChats } from "@liveblocks/react";
+
+function Chats() {
+  // +++
+  const { chats, error, isLoading } = useAiChats();
+  const [chatId, setChatId] = useState();
+  const deleteChat = useDeleteAiChat();
+  // +++
+
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  if (error) {
+    return <div>Error: {error.message}</div>;
+  }
+
+  return (
+    <div style={{ display: "flex" }}>
+      <ul>
+        // +++
+        {chats.map((chat) => (
+          <li key={chat.id}>
+            <button onClick={() => setChatId(chat.id)}>
+              {chat.title || "Untitled"}
+            </button>
+            <button onClick={() => deleteChat(chat.id)}>❌</button>
+          </li>
+        ))}
+        // +++
+      </ul>
+      // +++
+      <AiChat chatId={chatId} />
+      // +++
+    </div>
+  );
+}
+```
+
+#### Pagination [#useAiChats-pagination]
+
+By default, the `useAiChats` hook returns up to 50 chats. To fetch more, the
+hook provides additional fields for pagination, similar to [`useThreads`][].
+
+```tsx
+import { useAiChats } from "@liveblocks/react";
+
+const {
+  chats,
+  isLoading,
+  error,
+
+  +++
+  hasFetchedAll,
+  fetchMore,
+  isFetchingMore,
+  fetchMoreError,
+  +++
+} = useAiChats();
+```
+
+- `hasFetchedAll` indicates whether all available AI chats have been fetched.
+- `fetchMore` loads up to 50 more AI chats, and is always safe to call.
+- `isFetchingMore` indicates whether more AI chats are being fetched.
+- `fetchMoreError` returns error statuses resulting from fetching more.
+
+##### Pagination example [#useAiChats-pagination-example]
+
+The following example demonstrates how to use the `fetchMore` function to
+implement a “Load More” button, which fetches additional AI chats when clicked.
+The button is disabled while fetching is in progress.
+
+```tsx
+import { AiChat } from "@liveblocks/react-ui";
+import { useAiChats } from "@liveblocks/react/suspense";
+
+function Inbox() {
+  const { chats, hasFetchedAll, fetchMore, isFetchingMore } = useAiChats();
+
+  return (
+    <div>
+      {chats.map((chat) => (
+        <AiChat key={chat.id} chatId={chat.id} />
+      ))}
+      // +++
+      {hasFetchedAll ? (
+        <div>🎉 You're all caught up!</div>
```

Comment:
I think using "caught up" is a bit weird in the context of a chat list. It makes more sense in the context of inbox notifications. Maybe show "that's it" or something?